src/dom/__tests__/node-contains.test.ts

```diff
@@ -10,61 +10,52 @@ test('returns false if either node is null', () => {
   expect(nodeContains(null, null)).toBe(false);
 });
 
+// useful for passing `event.target` or `event.relatedTarget` without additional processing
+test('returns false if descendant is not a valid node', () => {
+  const div = document.createElement('div');
+  expect(nodeContains(div, window)).toBe(false);
+  expect(nodeContains(div, new EventTarget())).toBe(false);
+});
+
 const testCases: Record<string, string> = {
   regular: `<div id="parent">
-              <div id="inbetween">
-                <div id="child"></div>
-              </div>
-            </div>
+    <div id="inbetween">
+      <div id="child"></div>
+    </div>
+  </div>
 `,
-  svg: `<g id="parent">
```

Comment:
This test did not check what it says. `<g>` on its own is not an SVG element, only inside `<svg>`, it becomes special